sdk/src/main/java/com/bugsnag/android/DefaultDelivery.java

```diff
@@ -79,6 +79,9 @@ int deliver(String urlString,
             return conn.getResponseCode();
         } catch (IOException exception) {
             throw new DeliveryFailureException("IOException encountered in request", exception);
+        } catch (Exception exception) {
+            Logger.warn("Unexpected error delivering payload", exception);
+            return 400;
```

Comment:
This should be made a constant, to make it more declarative why `400` is being used here.